sorc/checkout.sh

```diff
@@ -22,16 +33,18 @@ while getopts "oc" option; do
  esac
 done
 
+ufs_checkout_hash=${ufs_checkout_hash:-${ufs_fv3gfs_hash}}
+
 topdir=$(pwd)
 echo $topdir
 
-echo ufs-weather-model checkout ...
+echo ufs-weather-model checkout revision ${ufs_checkout_hash} ...
 if [ ${COUPLED:-"NO"} = "NO" ]; then
   if [[ ! -d fv3gfs.fd ]] ; then
     rm -f ${topdir}/checkout-fv3gfs.log
     git clone https://github.com/ufs-community/ufs-weather-model fv3gfs.fd >> ${topdir}/checkout-fv3gfs.log 2>&1
     cd fv3gfs.fd
-    git checkout 9350745855aebe0790813e0ed2ba5ad680e3f75c 
+    git checkout ${ufs_checkout_hash}
```

Comment:
Shouldn't this one be: ufs_fv3gfs_hash

Reply:
ufs_checkout_hash is already set to ufs_fv3gfs_hash if no option is given.